templates/rest-info.json

```diff
@@ -0,0 +1,3 @@
+{
+	"currentVersion": 10.51
```

Comment:
should we set the version centrally so things stay in sync?

Reply:
Good call.  Added in 2aa8c41 and 4b94f3d.